app/grandchallenge/reader_studies/utils.py

```diff
@@ -17,27 +33,34 @@ def migrate_reader_study_to_display_sets(rs, view_content):  # noqa: C901
             ds = DisplaySet.objects.create(reader_study=rs)
             images = []
             for key in item:
-                image = rs.images.get(name=item[key])
-                images.append(image.pk)
-
+                if "overlay" in key:
+                    continue
```

Comment:
Why skip here?

Reply:
Overlays get handled alongside the main image for the viewport. The `*-overlay` key is no longer used in the view_content